azure-stack/hci/manage/manage-network-atc.md

```diff
@@ -129,19 +129,50 @@ This task will help you override the default configuration which has already bee
 
 If you want to test various configurations on the same adapters, you may need to remove an intent. 
 
-If you previously deployed and configured Network ATC on your system, you may need to reset the node so that the configuration can be deployed. To do this, copy and paste the following commands to remove all existing intents and their corresponding vSwitch:
+If you previously deployed and configured Network ATC on your system, you may need to reset the cluster or node so that the configuration can be deployed. To do this on a cluster-based deploynent, copy and paste the following commands to one cluster node to remove all existing intents and their corresponding vSwitch:
```

Comment:
I don't like the terminology of "reset the cluster" which I think gives the wrong impression. Consider: "you may need to remove the existing configuration as Network ATC will not remove them for you." or something to that effect.

" to one cluster node" should be "on one cluster node"